cypress/integration/homeFeedFlows/feedNavigation.spec.js

```diff
@@ -25,11 +25,6 @@ describe('Home Feed Navigation', () => {
         'aria-current',
         'page',
       );
-
-      cy.get('@week').should('have.attr', 'aria-current', '');
```

Comment:
There are a few more single-line expectations hiding in the other specs, with the empty attribute value, like: 

```javascript
cy.get('@infinity').should('have.attr', 'aria-current', '');
```